package.json

```diff
@@ -33,36 +33,40 @@
     "ember-test-selectors": "^7.0.0"
   },
   "devDependencies": {
+    "@babel/eslint-parser": "^7.25.1",
+    "@babel/plugin-proposal-decorators": "^7.25.9",
+    "@ember/test-helpers": "^2.6.0",
     "@ember/optional-features": "^2.2.0",
     "@glimmer/component": "^2.0.0",
     "@glimmer/tracking": "^1.1.2",
     "@release-it/conventional-changelog": "^2.0.1",
-    "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-cli": "~3.22.0",
+    "ember-cli": "~3.28.6",
     "ember-cli-dependency-checker": "^3.3.3",
-    "ember-cli-deploy": "^1.0.2",
-    "ember-cli-deploy-build": "^2.0.0",
+    "ember-cli-deploy": "^2.0.0",
+    "ember-cli-deploy-build": "^3.0.0",
     "ember-cli-deploy-git": "^1.3.4",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.1",
-    "ember-load-initializers": "^2.1.2",
+    "ember-load-initializers": "^3.0.1",
     "ember-maybe-import-regenerator": "^1.0.0",
-    "ember-qunit": "^4.6.0",
+    "ember-qunit": "^5.1.5",
```

Comment:
This is not the latest, but it's the latest that support Ember 3.28.
The v4 -> v5 upgrade also required a few changes to the tests/index.html and tests/test-helper.js files.